src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -401,4 +414,17 @@ private function checkUserAction(Entry $entry)
             throw $this->createAccessDeniedException('You can not access this entry.');
         }
     }
+
+    /**
+     * @return Pagerfanta
+     */
+    public function _countAllEntriesForUser()
+    {
+        $repository = $this->getDoctrine()->getRepository('WallabagCoreBundle:Entry');
+        $qb = $repository->getBuilderForAllByUser($this->getUser()->getId());
+        $pagerAdapter = new DoctrineORMAdapter($qb->getQuery());
+        $entries = new Pagerfanta($pagerAdapter);
```

Comment:
No, no no. Use a simple count SQL query with Doctrine (using the repo of course) we don't need to load the full pager etc for a simple count.